drafts/latest/tables/deprecated-properties-for-dataset.html

```diff
@@ -35,5 +36,7 @@
 
 <aside class="note">
 <p>Until GeoDCAT-AP version 1.0.1 [[GeoDCAT-AP-20160802]], property <code>rdfs:comment</code> was used with class <code>dcat:Dataset</code> to specify the spatial resolution of a Dataset, since a more specific property was not available in reference vocabularies.</p>
-<p>[[VOCAB-DCAT-2]] has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>, which has been adopted by [[DCAT-AP-20191120]]. Therefore, <code>dcat:spatialResolutionInMeters</code> replaces <code>rdfs:comment</code> starting from this version of GeoDCAT-AP, but only when spatial resolution is expressed as distance in metres.</p>
+<p>[[VOCAB-DCAT-2]] and [[?SDW-BP]] recommend the use of [[VOCAB-DQV]] for the specification of different types of spatial resolution, via property <code>dcat:hasQualityMeasurement</code>. Moreover, [[VOCAB-DCAT-2]] has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>.</p>
```

Comment:
[[VOCAB-DCAT-2]]  refers to DQV for expressing quality statements, but AFAIR, it does not explicitly refer to dqv:hasQualityMeasurement for spatial resolution.
Besides correcting  "dcat:hasQualityMeasurement" with "dqv:hasQualityMeasurement", I would consider rephrasing the note a little ...

```suggestion
<p>[[?SDW-BP]] recommends the use of [[VOCAB-DQV]] for the specification of different types of spatial resolution, via property <code>dqv:hasQualityMeasurement</code>.  [[VOCAB-DCAT-2]] reuses  [[VOCAB-DQV]]  for expressing quality statements and it  has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>.</p>
```

Reply:
Thanks again. I fixed the typo throughout the whole document.

Reply:
Yes, it works for me. Thanks.